requirements.txt

```diff
@@ -25,12 +25,12 @@ xmltodict==0.9.2
 -e git+https://git@github.com/uc-cdis/cdis-python-utils.git@0.2.8#egg=cdispyutils
 -e git+https://git@github.com/uc-cdis/userdatamodel.git@1.0.4#egg=userdatamodel
 -e git+https://git@github.com/uc-cdis/cdis_oauth2client.git@0.1.3#egg=cdis_oauth2client
--e git+https://git@github.com/uc-cdis/indexclient.git@1.5.5#egg=indexclient
+-e git+https://git@github.com/uc-cdis/indexclient.git@1.5.7#egg=indexclient
 -e git+https://git@github.com/uc-cdis/datamodelutils.git@0.4.0#egg=datamodelutils
 
 # required for gdcdatamodel, not required for sheepdog
 -e git+https://git@github.com/NCI-GDC/cdisutils.git@f54e393c89939b2200dfae45c6235cbe2bae1206#egg=cdisutils
--e git+https://git@github.com/uc-cdis/gdcdatamodel.git@1.3.0#egg=gdcdatamodel
+-e git+https://git@github.com/NCI-GDC/gdcdatamodel.git@dict_v1.15.0-Marvin#egg=gdcdatamodel
```

Comment:
I would run this by PlanX. I'm not sure we can just change gdcdatamodel like this.

Reply:
Yeah I revert this back. I think PlanX uses a different branch and doesn't seem like we are going to merge.. But since gdcapi will pin the gdcdatamodel to the correct one, so I'm leaving this unchanged.